wal/metrics.go

```diff
@@ -17,6 +17,13 @@ package wal
 import "github.com/prometheus/client_golang/prometheus"
 
 var (
+	syncGauge = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: "etcd",
+		Subsystem: "disk",
+		Name:      "wal_fsync_duration_current",
```

Comment:
Per Prometheus metric naming conventions the `_current` should be dropped. A gauge implies being "current".